configure.ac

```
@@ -14,7 +14,6 @@ AC_CHECK_HEADERS([stdlib.h string.h unistd.h sys/prctl.h])
 
 # Checks for library functions.
 AC_FUNC_FORK
-AC_FUNC_MALLOC
 AC_CHECK_FUNCS([alarm sqrt strerror])
```

It is still needed to check that a `malloc` function exists, it can be done in `AC_CHECK_FUNCS`.

```suggestion
AC_CHECK_FUNCS([alarm sqrt strerror malloc])
```